filament/backend/include/private/backend/HandleAllocator.h

```diff
@@ -24,22 +24,19 @@
 #include <utils/compiler.h>
 #include <utils/debug.h>
 #include <utils/ostream.h>
+#include <utils/Panic.h>
 
 #include <tsl/robin_map.h>
 
+#include <cstddef>
```

Comment:
is this needed even with `<stddef.h>` below?

Reply:
yes needed for `std::max_alight_t`